packages/static-assets/scripts/get-schema.ts

```diff
@@ -161,9 +161,7 @@ export function getSchema() {
         let children: string[] = [];
         let acceptsStringChildren = false;
 
-        // All components have the name and copySource attributes,
-        // even though they aren't in the attributes object
-        let attributes = [{ name: "name" }, { name: "copySource" }];
+        let attributes = [];
```

Comment:
Is `name` included in the schema now?

Reply:
Yeah, it was in there twice before. I hadn't realized that. But, I had added a `name` attribute to the base component in #318, which was causing `name` to go in twice!

Reply:
Okay. The rest looks good to me :-)